carbon-design-system/carbon
# fix(react): add prop adapter for downshift changes

Closes #6420 

Adds in the prop adapter for Downshift to map from `initial` to `default` without a breaking change

#### Changelog

**New**

**Changed**

- Add prop adapter to MultiSelect, Dropdown, Combobox, and Filterable MultiSelect

**Removed**

#### Testing / Reviewing

- Checkout the PR locally 
- Run storybook
- Update the combobox story with the example from: https://github.com/carbon-design-system/carbon/pull/6425 and verify that it works as intended
